packages/wonder-blocks-modal/components/modal-launcher-positioner.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+
+type Props = {
+    children: React.Node,
+    closeModal: () => void,
```

Comment:
This should probably be `onCloseModal` or even just `onClick`.  My preference is for the latter one b/c this component isn't the one doing the closing.

Reply:
Hmm yeah, maybe `onClickBackdrop`?